apps/ideaspace/src/components/modules/DashboardPage/Ideas/StyledIdeaCard.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+export const ImgButton = styled.button`
+  cursor: pointer;
+  border: 0 none;
+  padding: 0;
+  width: 22.5rem;
+`;
+
+export const StatuBox = styled.span`
+display: flex;
+flex-direction: row;
+justify-content: center;
+align-items: center;
+width: 120px;
+height: 40px;
+background: #C0E1EB;
+border-radius: 24px;
+z-index: 1;
+position: relative;
+left: 224px;
+top: -261px;
+
+font-family: 'Nunito Sans';
+font-style: normal;
+font-weight: 400;
+font-size: 16px;
```

Comment:
Is there any specific reason we are using `px` in this file and `rem` in other files in this PR?

Reply:
> Is there any specific reason we are using `px` in this file and `rem` in other files in this PR?

It not been used. It is preparation for status tag for next sprint.